### DISCUSSION TITLE
are Percent Symbol Space / Percent Symbol Placement (right or left) available?

### DISCUSSION THREAD

**Discussion by wd-l10n** [unicode-org/icu4x#4482] [Q&A]
Any pointers to where we could find / get these?
- Percent Symbol Space / Percent Symbol Placement (right or left)

if not available yet, could you tell us how we get them meanwhile?

**Comment by sffc** [comment]
They are not currently in the icu4x data bundles but would be fairly easy to add. I opened https://github.com/unicode-org/icu4x/issues/4483 to track this.
